supplementary_tests/CMakeLists.txt

```diff
@@ -1,10 +1,8 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(supplementary_tests)
 
-## Use c++ 11x std
-set(CMAKE_CXX_FLAGS "-std=c++11 ${CMAKE_CXX_FLAGS}")
 if (${CMAKE_EXTRA_GENERATOR} MATCHES "Eclipse CDT4")
-	set(CMAKE_CXX_COMPILER_ARG1 "-std=c++11" CACHE STRING "C++ version for eclipse" FORCE)
+	set(CMAKE_CXX_COMPILER_ARG1 "-std=c++14" CACHE STRING "C++ version for eclipse" FORCE)
```

Comment:
If you want to enforce c++14, this should be set regardless of which IDE is being used.

Reply:
That's done anyways in common config.

Reply:
Then do we need this line?